README.md

```diff
@@ -1 +1,100 @@
-# presto-js-client
\ No newline at end of file
+# Presto JS Client
+
+This is a Presto JavaScript client that connects to Presto via Presto's REST API to run queries.
+
+## Installation
+
+```sh
+npm install @presto/client
+```
+
+## Usage
+
+Import the PrestoClient class from `@presto/client`.
+
+Create a new instance by passing the connection parameters.
+
+### Example
+
+```typescript
+import { GetPrestoDataParams, PrestoClient } from '@presto/client'
```

Comment:
can the import be simplified to only `PrestoClient`? or do you plan to use `GetPrestoDataParams` somewhere?

Reply:
We can add a default import as well so if somebody does not need the types can use it directly

Reply:
some ideas:

we could have a default import and also have the types as named:
```typescript
import PrestoClient from '@prestodb/client'
import PrestoClient, { GetPrestoDataParams } from '@prestodb/client'
```

or make the PrestoClient be like a namespace and retrieve the types from there:
```typescript
import PrestoClient from '@prestodb/client'
...
function getClient(args: PrestoClient.GetPrestoDataParams) ...
```
although I'm not sure if it's possible for a class to have interfaces attached to it

Reply:
Def Option 1, is way more standard